hooks/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
aye, we're close to reinventing lerna, might as well just use it sooner rather than later